src/praisonai-code/praisonai_code/cli/interactive/frontends/rich_frontend.py

```
@@ -177,50 +182,56 @@ def _prompt_approval(self, request: ApprovalRequest) -> ApprovalResponse:
             
             console.print("[1] Allow once")
-            console.print("[2] Always allow this pattern")
-            console.print("[3] Always allow for this session")
-            console.print("[4] Reject")
+            console.print(f"[2] Always allow this command ({narrow_pattern})")
+            console.print(f"[3] Always allow this command for this session ({narrow_pattern})")
+            console.print(f"[4] Always allow ALL uses of {request.tool_name} ({blanket_pattern})")
+            console.print("[5] Reject")
             
         except ImportError:
             print(f"\n=== Approval Required ===")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)